cmd/globals.go

```diff
@@ -96,6 +99,12 @@ var (
 	// IsSSL indicates if the server is configured with SSL.
 	globalIsSSL bool
 
+	globalTLSCertificate *tls.Certificate
+
+	globalHTTPServer        *miniohttp.Server
+	globalHTTPServerErrorCh = make(chan error)
+	globalOSSignalCh        = make(chan os.Signal, 1)
```

Comment:
I understand calling `signal.Stop()` stops writing to the channel.  `signal.Notify()` is called in `serverMain()` after a goroutine starting http server and before object layer initialization.  This means, signals occur in between two codes also handled.

> All channels would still receive since kernel signal handler is common no matter how many signal.Notify you invoke.

Not sure I understand this correctly.